### PR TITLE
Updated docs with a small release guide

### PR DESCRIPTION
**Pull request by wojta**
### Motivation
To retain knowledge about releasing the operator within Integr8ly
### What
Changed README to cover this in section `Release`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,3 +114,22 @@ $ make setup/dep
...
+
+1. Bump the version number (e.g. `0.0.29`) (in more files [Makefile](https://github.com/integr8ly/application-monitoring-operator/blob/master/Makefile#L6), [deploy/operator.yaml](https://github.com/integr8ly/application-monitoring-operator/blob/master/deploy/operator.yaml), [version/version.go](https://github.com/integr8ly/application-monitoring-operator/blob/master/version/version.go#L4]))
+2. Create pull request with the bump, let it be approved and merge it to `master`
+3. Pull the master locally and tag it with a tag named as the version number 
+```sh
+$ git pull origin master
+$ git tag 0.0.29
+$ git push origin --tags
+```
+4. Create *Release* through github release page, name it as the version number - `0.0.29` and state what's new in the release
```
Comment (david-martin) [inline README.md]: @wojta Will creating a release through github release page also create the tag? That would remove the need for steps 2 & 3.

Comment (david-martin) [reply README.md]: ping @wojta for q above

Comment (wojta) [reply README.md]: Sorry, completely forgot this. 
Yes it does make the tag. Fixing now.

**Comment by david-martin** [comment]
superceded by #106

**Pull request closed without merging**